content/post/67-crdt-research-meetup.md

```diff
@@ -0,0 +1,60 @@
+---
+date: 2019-01-28
+url: 67-crdt-research-meetup
+tags: crdt research meetup videos ipfs
+title: 2018 CRDT Research Meetup - Lisbon
+author: Pedro Teixeira
+---
+
+On April last year some members of the IPFS team gathered in Lisbon for a week of fun hack. During this week we mostly hacked and collaborated on some aspects of IPFS, libp2p and applications built on top of them, but also took one day to host a CRDT Research meetup.
+
+CRDT stand for Conflict-free Replicated Data Types and are a mathematical framework and building block that allows different replicas to cooperate without requiring coordination, making them very useful for creating eventually consistent data layers over intermittent and unreliable networks. For instance, CRDTs make sure that, no matter the order of the messages exchanged, all the peers eventually converge to the same state.
+
+([You can read more about CRDTs here](https://github.com/ipfs/research-CRDT))
+
+So, taking the opportunity of having some of The IPFS team in Lisbon, we had the pleasure of hosting some researchers in this field for one day of presentations and discussions. Here are some videos of some of the sessions that happened during this meetup. Enjoy!
```

Comment:
```suggestion
We believe that CRDTs are one fundamental building block for building completely Decentralized, Distributed, Offline First Web Applications and the [IPFS Dynamic Data & Capabilities Working Group](http://github.com/ipfs/dynamic-data-and-capabilities) has been spearheading the development and pushing IPFS and libp2p with projects such as: [PeerPad](http://peerpad.net/), [PeerBase](https://github.com/peer-base), [Discussify](https://github.com/ipfs-shipyard/discussify-browser-extension) and the [Sandboxed Identity Manager Project](https://github.com/ipfs-shipyard/pm-idm).

If you didn't have the chance to attend this meetup in person, you can now follow what happened by checking the talks below. Enjoy!
```

Reply:
"checking the talks below, or browsing through [this playlist](https://www.youtube.com/playlist?list=PLuhRWgmPaHtTVkko1ZTn-qcGb-n6EqHff)."